doc/modules/ROOT/pages/pipelines.adoc

```diff
@@ -4,10 +4,9 @@
 The Python client has special support for https://neo4j.com/docs/graph-data-science/current/machine-learning/linkprediction-pipelines/link-prediction/[Link prediction pipelines] and https://neo4j.com/docs/graph-data-science/current/machine-learning/node-property-prediction/[pipelines for node property prediction].
 The GDS pipelines are represented as pipeline objects.
 
-The Python method calls to _create_ the pipelines match their Cypher counterparts exactly.
-However, the rest of the pipeline functionality is deferred to methods on the pipeline objects themselves.
-Once created, the `TrainingPipeline` can be passed as arguments to methods in the Python client, such as the https://neo4j.com/docs/graph-data-science/current/pipeline-catalog/pipeline-catalog/[pipeline catalog operations].
-Additionally, the `TrainingPipeline` has convenience methods allowing for inspection of the pipeline represented without explicitly involving the pipeline catalog.
+While it is possible to operate pipelines using the same signatures as the Cypher procedures, the GDS Python Client offers a more convenient way by using the pipeline objects.
```

Comment:
This is not actually true I think. Only the create procedures are mapped to Python methods. Unless you mean by using `run_cypher`?

Reply:
No I meant by mimicking -- will change this sentence.